[2024-08-02T14:23:17.958+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-02T14:23:17.968+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_supermarket_2.copy_table_dim_aisles scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-08-02T14:23:17.975+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_supermarket_2.copy_table_dim_aisles scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-08-02T14:23:17.975+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-02T14:23:18.000+0700] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): copy_table_dim_aisles> on 2024-08-01 00:00:00+00:00
[2024-08-02T14:23:18.002+0700] {standard_task_runner.py:64} INFO - Started process 14535 to run task
[2024-08-02T14:23:18.004+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline_supermarket_2', 'copy_table_dim_aisles', 'scheduled__2024-08-01T00:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_supermarket_2.py', '--cfg-path', '/tmp/tmpnu3ghcd1']
[2024-08-02T14:23:18.005+0700] {standard_task_runner.py:91} INFO - Job 130: Subtask copy_table_dim_aisles
[2024-08-02T14:23:18.042+0700] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_supermarket_2.copy_table_dim_aisles scheduled__2024-08-01T00:00:00+00:00 [running]> on host DESKTOP-RHVHV7A.
[2024-08-02T14:23:18.115+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline_supermarket_2' AIRFLOW_CTX_TASK_ID='copy_table_dim_aisles' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-01T00:00:00+00:00'
[2024-08-02T14:23:18.115+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-02T14:23:18.125+0700] {sql.py:282} INFO - Executing: 
        COPY dim_aisles TO '/mnt/f/AllCode/Data Engineer/dim_aisles.csv' CSV HEADER;
        
[2024-08-02T14:23:18.128+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-02T14:23:18.145+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-02T14:23:18.177+0700] {sql.py:495} INFO - Running statement: 
        COPY dim_aisles TO '/mnt/f/AllCode/Data Engineer/dim_aisles.csv' CSV HEADER;
        , parameters: None
[2024-08-02T14:23:18.179+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-02T14:23:18.179+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/operators/sql.py", line 288, in execute
    output = hook.run(
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 442, in run
    self._run_command(cur, sql_statement, parameters)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 500, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "dim_aisles" does not exist

[2024-08-02T14:23:18.184+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=data_pipeline_supermarket_2, task_id=copy_table_dim_aisles, run_id=scheduled__2024-08-01T00:00:00+00:00, execution_date=20240801T000000, start_date=20240802T072317, end_date=20240802T072318
[2024-08-02T14:23:18.204+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 130 for task copy_table_dim_aisles (relation "dim_aisles" does not exist
; 14535)
[2024-08-02T14:23:18.217+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-02T14:23:18.225+0700] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-08-02T14:23:18.236+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-02T14:23:18.236+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
